packages/grafana-ui/src/components/DataSourceSettings/types.ts

```diff
@@ -14,4 +14,6 @@ export interface HttpSettingsProps extends HttpSettingsBaseProps {
   showAccessOptions?: boolean;
   /** Show the SigV4 auth toggle option */
   sigV4AuthToggleEnabled?: boolean;
+  /** Hide the Forward OAuth identity option */
+  hideForwardOAuthIdentityOption?: boolean;
```

Comment:
I think its easier to reason about if the boolean prop is the _positive_ of the option.

Instead, there should be `showForwardOAuthIdentityOption` toggle, which can default to whatever you want.

Reply:
Spoke with @joshhunt in Slack and this approach shouldn't be a breaking change with this naming option AFAIK